pkg/trace/traceutil/azure.go

```diff
@@ -71,27 +71,46 @@ func getAppServicesTags(getenv func(string) string) map[string]string {
 		aasSiteType:        appService,
 		aasSubscriptionID:  subscriptionID,
 	}
+
+	// Remove the Java and .NET logic once non-universal extensions are deprecated
+	if websiteOS == "windows" {
+		if len(extensionVersion) > 0 {
+			tags[aasExtensionVersion] = extensionVersion
+		} else if hasEnv("DD_AAS_JAVA_EXTENSION_VERSION", getenv) {
+			tags[aasExtensionVersion] = getenv("DD_AAS_JAVA_EXTENSION_VERSION")
+		} else if hasEnv("DD_AAS_DOTNET_EXTENSION_VERSION", getenv) {
+			tags[aasExtensionVersion] = getenv("DD_AAS_DOTNET_EXTENSION_VERSION")
+		}
+	}
+
+	return tags
 }
 
 func getEnvOrUnknown(env string, getenv func(string) string) string {
 	val := getenv(env)
-	if len(env) == 0 {
+	if len(val) == 0 {
```

Comment:
I think this is a bug because if we're checking `len(env)` that's the length of the **name** of the env var rather than the actual **value**

Reply:
Good catch!